src/lib/init-hooks/core-hooks/http-logger.js

```diff
@@ -0,0 +1,50 @@
+const url = require('url');
+
+module.exports = function(BoringInjections) {
+
+  const {
+    boring,
+    logger,
+    config,
+  } = BoringInjections;
+
+  boring.before('add-hooks', function() {
+
+    boring.before('app.use', function(ctx) {
+      if (ctx.name === 'context') {
+        boring.app.use('http-logger', function loggerMiddleware(req, res, next) {
+
+          const parsedUrl = url.parse(req.originalUrl);
+          const path = (parsedUrl.pathname || '').toLowerCase();
+          const ext = path.split('.').pop();
+          const logLevel = (path.indexOf('_hmr')>0 ||
```

Comment:
I've learned to NOT log static assets cause it fills up your logs with junk real fast.  They are still in trace if you really want to see them.